Source/Dafny/Verifier/Translator.cs

```diff
@@ -44,6 +44,7 @@ public partial class Translator {
     public class TranslatorFlags {
       public bool InsertChecksums = 0 < DafnyOptions.O.VerifySnapshots;
       public string UniqueIdPrefix = null;
+      public bool ReportRanges = false;
```

Comment:
When do we not want to report ranges? Looks like it's turned on for the IDE but not the CLI. Why the difference?

Reply:
Yes, because the IDE will highlight ranges. However, in the CLI, there is a format to report positions only, so that would mean 1) to change that format to support ranges of positions, which would be cluttering 2) or we could also decide so activate this option when -showSnipptes is enabled, but it's not the option by default yet and it will require to change the way we display errors anyway.

I though we should first add these ranges in the Dafny Language Server, where they are the most useful, and then we can always do this later.